src/openforms/logging/tests/test_events.py

```diff
@@ -33,3 +38,42 @@ def test_submission_start(self):
 
             log = TimelineLogProxy.objects.last()
             self.assertEqual(_("Anonymous user"), log.fmt_user)
+
+    def test_submission_logic_evaluated(self):
+        submission = SubmissionFactory.create()
+        form = FormFactory.create()
+        form_step = FormStepFactory.create(form = form)
+        submission_step = SubmissionStepFactory.create(submission = submission, form_step = form_step)
```

Comment:
You can cut down a lot of boilerplate with `SubmissionFactory.from_data` or `SubmissionFactory.from_components`